sources/web/datalab/server.ts

```diff
@@ -209,12 +238,18 @@ export function run(settings: common.Settings): void {
   infoHandler = info.createHandler(settings);
   staticHandler = static.createHandler(settings);
 
-  server = http.createServer(requestHandler);
```

Comment:
Hopefully the changes in this file are not needed, once we get rid of the initialization of a datastore wrapper.